addons/web/static/src/js/core/utils.js

```diff
@@ -1002,6 +1002,22 @@ var utils = {
             ['name', '=like', '%.assets\_%.js'],
         ];
     },
+    /**
+     * Merge multiple objects props ( including Getters and Setters )
+     * @param  {(Object|...Object)} sources
+     * @returns {Object} New object with sources objects props merged
+     */
+    mix : function (...sources) {
```

Comment:
Let's see what @aab-odoo  would say,
In my opinion a goo'ol' object.create/object.assign is better, less magic, more straightforward than an obscure utility that we'd have to maintain

This is generally related to how we do class inheritance in a owl context
So pinging @mcm-odoo @ged-odoo ; just to think about it in the patch method PR